src/api/subscriptions.js

```diff
@@ -116,6 +116,12 @@ export class Subscriptions {
    */
   setOnLoginRequest(callback) {}
 
+  /**
+   * Starts the LoginPrompt flow.
+   * @return {!Promise}
+   */
+  loginPrompt() {}
```

Comment:
We are currently using verbs for action methods. So, either `promptLogin` or `showLoginPrompt`?

Reply:
done